Tests/UtilityTests/processInputs/print-pid

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo $$
+yes > /dev/null
```

Comment:
Given this test isn't using `yes`, it would be safer to write: `while true; do sleep 1; done`

This script should also start with: `set -e` to ensure it exits immediately whenever any of the subtasks has a problem. Without this, there is a race if the `echo` gets the SIGINT (which may be why you needed the sleep).